src/core/unit.h

```diff
@@ -879,6 +879,11 @@ static inline bool unit_has_job_type(Unit *u, JobType type) {
         return u && u->job && u->job->type == type;
 }
 
+static inline bool unit_filters_log_level(const Unit *u, int level) {
+        ExecContext *ec = unit_get_exec_context((Unit *)u);
+        return ec && ec->log_level_max >= 0 && ec->log_level_max < LOG_PRI(level);
+}
```

Comment:
i'd invert the function result btw. i.e. i'd always focus on the positive case, i.e. letting the logging go though. i.e. invert and rename unit_log_level_test(…) or so?